build.zig

```diff
@@ -542,6 +550,7 @@ pub inline fn makeCubyzLibs(b: *std.Build, step: *std.Build.Step, name: []const
 	c_lib.installHeader(b.path("include/stb/stb_image.h"), "stb/stb_image.h");
 	c_lib.installHeader(b.path("include/stb/stb_vorbis.h"), "stb/stb_vorbis.h");
 	c_lib.installHeader(b.path("include/miniaudio.h"), "miniaudio.h");
+	addCGltf(b, c_lib, flags);
```

Comment:
Instead of making a new function for each header only library, how about making one function for all of them, since the setup is usually quite small.